### PR TITLE
Creating Cmdlets for Alert Rules (Insights)

### PR DESCRIPTION
**Pull request by gucalder**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@gucalder__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (gucalder). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by ogail** [comment]
@gucalder please add at least one mocked scenario test associated with these cmdlets

File: src/ResourceManager/Insights/Commands.Insights/Alerts/AddAlertRuleCommand.cs [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+                case AlertRuleTypes.Webtest:
+                    condition = new LocationThresholdRuleCondition()
+                    {
+                        DataSource = new RuleMetricDataSource(),
+                        FailedLocationCount = this.FailedLocationCount,
+                        WindowSize = this.WindowSize,
+                    };
+                    break;
+                default:
+                    throw new NotSupportedException(string.Format(CultureInfo.InvariantCulture, "Rule type {0} is not supported", this.RuleType));
```
Comment (amarzavery) [inline src/ResourceManager/Insights/Commands.Insights/Alerts/AddAlertRuleCommand.cs]: Please add the text message in Resources.Resx and use it from there

**Comment by amarzavery** [comment]
@gucalder - Please address the review comments so that we can merge the PR and make it a part of this PS release.

**Comment by markcowl** [comment]
@gucalder Please resolve issues and retarget at the release-0.8.15 branch

**Pull request closed without merging**